hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/update/UpdateTranslator.java

```diff
@@ -83,8 +91,10 @@ private TransactionBody nominalBodyFor(@NonNull final HtsCallAttempt attempt) {
             return decoder.decodeTokenUpdateV1(attempt);
```

Comment:
It seems using the streams may make this nicer to look at (see CreateTranslator)?

Reply:
Added the stream version for update as well